lib/widgets/pages/tickets/analog_closed_popup.dart

```diff
@@ -0,0 +1,59 @@
+part of 'tickets_page.dart';
```

Comment:
Sections aren't pages, but components.
Move file out of pages directory

Reply:
To be fair, I see the reasoning of this file structure, but we might have to look at folder structure generally if we want all Tickets-only widgets components and page widget to be in one folder.